.golangci.yml

```diff
@@ -15,7 +15,12 @@ issues:
     - path: "(central|compliance|integration-tests|local|migrator|operator|pkg|sensor|tests|tools|scale)/"
       linters:
         - wrapcheck
+    - path: "(central/graphql/schema/print|compliance|integration-tests|local|migrator|operator|pkg|sensor|tests|tools|scale)/"
```

Comment:
I think we should not exclude sensor. If there are `fmt.Print`, we should change them to `errors.Info/Warn...`.  Nonetheless, it can come in a follow-up PR.